evm_loader/program/src/account/mod.rs

```diff
@@ -159,6 +161,31 @@ where
         Ok(Self { dirty: false, data, extension, info })
     }
 
+    #[deprecated]
+    fn reinit(info: &'a AccountInfo<'a>, data: T) -> Result<Self, ProgramError> {

```

Comment:
Good point. But not replace: we need a constructor here. This works (init instead of reinit):
```
        let info = v1.info;
        info.data.borrow_mut()[0] = TAG_EMPTY;
        Self::init(info, data)
```